modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -329,6 +331,43 @@ public static void exitNestedEventLoop(Object key, Object rval) {
         Toolkit.getToolkit().exitNestedEventLoop(key, rval);
     }
 
+    /**
+     * Returns a flag indicating whether the key corresponding to {@code keyCode}
+     * is in the locked or "on" state.
+     * {@code keyCode} must be one of: {@link KeyCode#CAPS} or
+     * {@link KeyCode#NUM_LOCK}.
+     * If the underlying system is not able to determine the state of the
+     * specified {@code keyCode}, an empty {@code Optional} is returned.
+     * If the keyboard attached to the system doesn't have the specified key,
+     * {@code Boolean.FALSE} is returned.
```

Comment:
Not `Optional.of(false)`?

Reply:
Yes, that would be more accurate. So the replacement could be either `{@code Optional.of(false)}` or `an @{code Optional} containing {@code false}`. At least one JDK method uses something like the latter, so I'll probably go with that.